mempool/src/core_mempool/mempool.rs

```diff
@@ -303,6 +303,13 @@ impl Mempool {
             aptos_infallible::duration_since_epoch_at(&now) + self.system_transaction_timeout;
 
         let sender = txn.sender();
+        counters::SENDER_BUCKET_FREQUENCIES
```

Comment:
Interesting place to put this. Would it be better to count below if the txn is actually accepted? (`if status.code == MempoolStatusCode::Accepted`)

Reply:
Done.